src/Linfo/Linfo.php

```diff
@@ -87,7 +87,17 @@ public function __construct($settings = array())
         $distro_class = '\\Linfo\\OS\\'.$os;
         $this->parser = new $distro_class($this->settings);
     }
-
+    
+    // Forward missing method request to the parser
+    public function __call ( $name , $args )
+    {
+        if (method_exists($this->parser, $name) && is_callable ( array($this->parser, $name))) {
```

Comment:
Can you remove the extra space before array(. Eg:

    if (method_exists($this->parser, $name) && is_callable (array($this->parser, $name))) {